modules/tinymce/src/core/main/ts/api/ui/Registry.ts

```diff
@@ -220,6 +220,19 @@ const registry = (): Registry.Registry => {
      */
     addToggleMenuItem: bridge.addToggleMenuItem,
 
+    /**
+     * Registers a new view container.
+     * This view container is hidden by default and attached next to the main view
+     * and can be toggled on or off. When it's toggled on the main editor view is hidden and the
+     * specific view is shown in it's place when toggled off the specific view is hidden and the main view becomes visible again.
+     * There is a ToggleView command that can be used to toggle the visibility of the view on or off or queried for the current state.
+     *
```

Comment:
```suggestion
    /**
     * Registers a new view container.
     * This view container is hidden (off) by default and attached next to the main view.
     * It can be toggled on or off.
     * When it is on, the main editor view is hidden and the specific view is shown.
     * When it is off, the specific view is hidden and the main view is shown.
     * There is also a ToggleView command.
     * The ToggleView command can toggle the view visibility.
     * The ToggleView command can also query the view for its current state.
```
Edited to be simple, declarative sentences.

Reply:
Changed the last sentence to `The ToggleView command can be queried for its current state.` not sure how to describe that. Since if you have `The ToggleView command can also query the view for its current state.` it could be interpreted that the command that is being executed can query things. Maybe the command stuff should not even be in here but they are for the sidebar api docs and that is what I used as a reference here so have a similar style.

Reply:
FWIW, your re-write of the last sentence is better (for me, at least) for another reason.

My suggested re-write —

```
The ToggleView command can also query the view for its current state.
```

— has the command querying another object — the `view`.

Your re-write —

```
The ToggleView command can be queried for its current state.
```

Has the command being queried.

In linguistic terms, my version makes ToggleView the subject (the thing doing). Your version makes ToggleView the object (the thing being done to).

That’s a straight up reversal, which means my re-write was based on a complete misunderstanding of the original.

Which further suggests the original was very easy to mis-interpret.

All of which is a strong argument for your re-write.